frame/identity/src/lib.rs

```diff
@@ -74,285 +74,22 @@
 
 #[cfg(test)]
 mod tests;
+mod types;
 mod benchmarking;
 pub mod weights;
 
 use sp_std::prelude::*;
-use sp_std::{fmt::Debug, ops::Add, iter::once};
-use enumflags2::BitFlags;
-use codec::{Encode, Decode};
-use sp_runtime::RuntimeDebug;
+use sp_std::convert::TryInto;
 use sp_runtime::traits::{StaticLookup, Zero, AppendZerosInput, Saturating};
-use frame_support::traits::{Currency, ReservableCurrency, OnUnbalanced, BalanceStatus};
+use frame_support::traits::{BalanceStatus, Currency, OnUnbalanced, ReservableCurrency};
 pub use weights::WeightInfo;
 
 pub use pallet::*;
+use types::*;
```

Comment:
Please enumerate the types imported instead of using a glob. It makes it easier to read later, so we can discover where a particular type comes from, particularly when it's not defined in this file.